src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2227,6 +2228,7 @@ private function registerNotifierConfiguration(array $config, ContainerBuilder $
             SendinblueNotifierTransportFactory::class => 'notifier.transport_factory.sendinblue',
             DiscordTransportFactory::class => 'notifier.transport_factory.discord',
             LinkedInTransportFactory::class => 'notifier.transport_factory.linkedin',
+            GatewayAPITransportFactory::class => 'notifier.transport_factory.gatewayapi',
```

Comment:
I would prefer:
```diff
- GatewayAPITransportFactory
+ GatewayApiTransportFactory
```

Same for all the others, but lets wait for @fabpot meaning on this

Reply:
The official name of the service is GatewayAPI, I prefer GatewayApiTransportFactory too.

Reply:
I agree with Oskar, let's be consistent with the rest of the code base.

Reply:
Done the refactor. I have maintained GatewayAPI only in `composer.json`, `README.md` and `phpunit.xml.dist` when the official name is provided, like:

`<testsuite name="Symfony GatewayAPI Notifier Bridge Test Suite">`

Reply:
Thank you 👍

Reply:
Thank you @OskarStark ❤️